src/genie/libs/parser/iosxr/tests/ShowImDampeningIntf/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,14 @@
+expected_output = {
+            'interface': {
+                'GigabitEthernet0/2/0/0': {
+                    'currently_suppressed': 'no',
```

Comment:
let's add 'interface_handle` here.
```
'GigabitEthernet0/2/0/0': {
    'interface_handle`: '0x080002c0',
```
corresponding output:
```
GigabitEthernet0/2/0/0 (0x080002c0)

Reply:
Certainly, easy to add that into the schema.

Reply:
confirmed the change